src/playground/sample_data_ingestion/sample_data_ingestor/sample_data_ingestor.py

```diff
@@ -0,0 +1,345 @@
+from datetime import datetime, timedelta
+from shutil import copyfileobj
+from json import loads, dumps
+from os import remove, path
+from random import randint
+import gzip
+import sys
+import csv
+
+# Adds parent directory "/sample_data_ingestion" to sys.path
+sys.path.append(path.dirname(path.dirname(path.abspath(__file__))))
+from sample_data_generator.sample_data_generator import generate_data
+from sample_data_generator.sample_data_generator import choose_field
+
+
+def generate_noise(initial_value, anomaly_percentage, avg_min, avg_max, abs_min, abs_max):
```

Comment:
do you have some logic to valid all the args here? thanks

Reply:
Yes, I revised `ingest_validation` to validate all possible arguments.